src/components/Post/index.jsx

```diff
@@ -15,15 +16,19 @@ export default function PostVolunteer(postInfo) {
   ) {
     return (
       <Col className="singlePost" lg={3} sm={12} md={6}>
-        <p className="cardTitle text-center">{postInfo.postInfo.title}</p>
-        <p className="postDate">Date: {postInfo.postInfo.date}</p>
-        <p className="postName">By : {postInfo.postInfo.profileName}</p>
-        <p className="cardBody text-center">{postInfo.postInfo.description}</p>
-        <img
-          className="cardImg"
-          src={postInfo.postInfo.image}
-          alt={t("postPage.altVolunteerHeader")}
-        />
+        <Link to={"posts/" + postInfo.postInfo.id}>
```

Comment:
It’s more convenient to write the link this way,  `<Link to=`posts/${postInfo.postInfo.id}` >`